include/qpdf/Pl_DCT.hh

```diff
@@ -34,6 +34,11 @@ class QPDF_DLL_CLASS Pl_DCT: public Pipeline
     QPDF_DLL
     Pl_DCT(char const* identifier, Pipeline* next);
 
+    // Constructor for decompressing image data. If corrupt_data_limit is non-zero and the data is
+    // corrupt, only attempt to uncompress if the uncompressed size is less than corrupt_data_limit.
+    QPDF_DLL
+    Pl_DCT(char const* identifier, Pipeline* next, size_t corrupt_data_limit);
```

Comment:
Usually when I add a new constructor whose first arguments overlap an existing one, I add an ABI comment reminding me to make the new argument optional for the next ABI window, but it doesn't matter.